### ISSUE TITLE
Connects and then terminates

### ISSUE THREAD

**Issue by theantihero** [adamncsu/discobot-pogo#1]
It connects fine and then suddenly terminates with no error.
![image](https://cloud.githubusercontent.com/assets/11086148/17446842/434d00f4-5b19-11e6-9ab5-467dc60458f0.png)

**Comment by adamncsu** [comment]
Yeah, I think its a permissions issue for winston. You need to manually the 'logs' directory, and then winston will be able to write logs there.

**Comment by theantihero** [comment]
ah ic thank you
